packages/vue-components/src/Navbar.vue

```diff
@@ -160,16 +165,25 @@
             if (hlMode === 'sibling-or-child') {
               if (this.isSibling(url, a.href) || this.isChild(url, a.href)) {
                 li.classList.add('current');
+                if (dropdownLinks.includes(a)) {
```

Comment:
let's move L167-170 to a inner function and use that - less repitition

Reply:
Alright thanks. I have shifted this to an inner function.